CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.4.1] - 2022-03-12
+
+### Changed
+
+- Changed logo text color from #333 to #44533f
```

Comment:
This change does not need to be in the Changelog, please reset this file to its original state.